docproc/src/main/java/com/yahoo/docproc/jdisc/messagebus/MbusRequestContext.java

```diff
@@ -96,12 +97,16 @@ public void processingDone(List<Processing> processings) {
         }
         long inputSequenceId = requestMsg.getSequenceId();
         ResponseMerger responseHandler = new ResponseMerger(requestMsg, messages.size(), this);
+        int numMsgWithOriginalSequenceId = 0;
         for (Message message : messages) {
+            if (message.getSequenceId() == inputSequenceId) numMsgWithOriginalSequenceId++;
+        }
+        for (Message message : messages) {
+            String path = internalNoThrottledSourcePath;
+            if ((numMsgWithOriginalSequenceId == 1) && (message.getSequenceId() == inputSequenceId))
```

Comment:
Shouldn't this be numMsgWithOriginalSequenceId >= 1 ?

Reply:
No, you shall only continue on the original path if there is exactly one operation with the same id as the incoming operation. If there are multiple operations with the same id they must always be sequenced. If they are to the original id or not does not matter.

Reply:
The if is actually the normal path :)

Reply:
👍